common/extensions/api/brave_rewards.json

```diff
@@ -369,6 +369,172 @@
         "description": "Creates new wallet",
         "parameters": []
       },
+      {
+        "name": "updateMediaDuration",
+        "type": "function",
+        "description": "Update the media duration for a given resource",
+        "parameters": [
+          {
+            "name": "publisherKey",
+            "type": "string"
+          },
+          {
+            "name": "duration",
+            "type": "integer"
```

Comment:
is greaselion sending us double or int?

Reply:
Greaselion sends an int (the duration is calculated as a double and then rounded to an int using `Math.round`, similar to what we previously did in C++).